ee/codegen/src/__test__/project.test.ts

```diff
@@ -1471,6 +1471,124 @@ baz = foo + bar
       expectProjectFileToMatchSnapshot(["code", "nodes", "first_node.py"]);
     });
   });
+  describe("Nodes with custom module ending in alpha and node reference", () => {
+    const firstNodeId = uuidv4();
+    const secondNodeId = uuidv4();
+    const firstNodeOutputId = uuidv4();
+    const firstNodeDefaultPortId = uuidv4();
+    const firstNodeTriggerId = uuidv4();
+    const secondNodeTriggerId = uuidv4();
+
+    const displayData = {
+      workflow_raw_data: {
+        nodes: [
+          {
+            id: "entry",
+            type: "ENTRYPOINT",
+            data: {
+              label: "Entrypoint",
+              source_handle_id: "entry_source",
+              target_handle_id: "entry_target",
+            },
+            inputs: [],
+          },
+          {
+            id: firstNodeId,
+            type: "GENERIC",
+            label: "beta",
+            attributes: [],
+            outputs: [
+              {
+                id: firstNodeOutputId,
+                name: "output",
+                type: "STRING",
+              },
+            ],
+            ports: [
+              {
+                id: firstNodeDefaultPortId,
+                name: "default",
+                type: "DEFAULT",
+              },
+            ],
+            trigger: {
+              id: firstNodeTriggerId,
+              merge_behavior: "AWAIT_ATTRIBUTES",
+            },
+            base: {
+              name: "BaseNode",
+              module: ["vellum", "workflows", "nodes", "alpha"],
+            },
```

Comment:
this should be the `definition` field. The `base` field should be the base node, like what the second node does below